src/cpp/src/visual_language/pipeline.cpp

```diff
@@ -611,6 +645,7 @@ class ov::genai::VLMPipeline::VLMPipelineImpl {
         }
         images_prompt += prompt;
         ov::Tensor encoded_input;
+        ov::Tensor new_chat_tokens;
```

Comment:
looks like we can return declaration of this variable inside `if` statement as it as before